src/stats/stats_printer.cpp

```diff
@@ -69,6 +69,33 @@ typedef enum {
 	e_M = 1048576
 } units_t;
 
+
+// this enum is copied from tcp.c in lwip project
+enum tcp_state {
+	CLOSED      = 0,
+	LISTEN      = 1,
+	SYN_SENT    = 2,
+	SYN_RCVD    = 3,
+	ESTABLISHED = 4,
+	FIN_WAIT_1  = 5,
+	FIN_WAIT_2  = 6,
+	CLOSE_WAIT  = 7,
+	CLOSING     = 8,
+	LAST_ACK    = 9,
+	TIME_WAIT   = 10
+};
+
+// this enum is copied from tcp.c in lwip project
+enum proc_net_type {
```

Comment:
1. Better not to copy definitions from other files (if possible)
2. It seems all you need is "TX" versus "RX" distinction. If so - please remove this copied enum and define you own, something like
```code
enum direction_type {
DIRECTION_TX,
DIRECTION_RX
} ;
```